frontend/ui/src/components/projects/roadmap-view/components/calendar-header.tsx

```
@@ -0,0 +1,72 @@
+  const isToday = (date: DateTime) => {
+    return date.hasSame(today, "day")
+  }
+  console.log(view, columnWidth)
+
+  return (
+    <Box className="sticky top-0 z-20 h-16 border-gray-5 border-b bg-white">
+      <Grid height="full" columns="min-content" flow="column">
+        {view === "year"
+          ? dates.map((date, index) => (
+              <Box
+                key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)